site/_data/templates.yaml

```diff
@@ -127,6 +130,36 @@
     - name: user
       type: User
       description: The User object corresponding to the authenticated user.
+# Account add WebAuthn passkey
+- displayName: Account add WebAuthn passkey
+  fieldName: accountWebAuthnAdd
+  description: This page contains a form that allows a user to register a new WebAuthn passkey.
+  path: /account/webauthn/add
+  version: 1.41.0
+  variables:
+    - name: user
+      type: User
+      description: The User object corresponding to the authenticated user.
+# Account delete WebAuthn passkey
+- displayName: Account delete WebAuthn passkey
+  fieldName: accountWebAuthnDelete
+  description: This page contains a form that allows a user to delete a WebAuthn passkey.
+  path: /account/webauthn/delete
+  version: 1.41.0
+  variables:
+    - name: credential
+      type: WebAuthnCredential
+      description: The WebAuthn passkey the user is deleting.
+# Account WebAuthn index
+- displayName: Account WebAuthn index
+  fieldName: accountWebAuthnIndex
+  description: This page displays an authenticated user's registered WebAuthn passkeys. Additionally, it provides links to delete an existing passkey and register a new passkey.
+  path: /account/webauthn/
+  version: 1.41.0
+  variables:
+    - name: webAuthnCredentials
+      type: List<Object>
+      description: The list of registered WebAuthn passkeys for the current user.
```

Comment:
Other variables have a link to API response documentation that shows what the objects look like. These don't exist for WebAuthn yet. I made a doc task to circle back and add the link.

Reply:
Have a doc task to add the link once I've created WebAuthn API docs.